src/coreclr/vm/ilstubcache.h

```diff
@@ -53,6 +53,8 @@ class ILStubCache final
         ILStubHashBlob* pHashBlob,
         DWORD dwStubFlags,      // bitmask of NDirectStubFlags
         Module* pSigModule,
+        Module* pSigLoaderModule,
+        SigTypeContext* pTypeContext,
```

Comment:
It would be nice to be a bit more consistent on where the `SigTypeContext` exists for these things. If I recall most times the `SigTypeContext` is after the `PCCOR_SIGNATURE` and length.